build-system/tasks/extension-generator/index.js

```diff
@@ -303,9 +295,9 @@ module.exports = {
 
 makeExtension.description = 'Create an extension skeleton';
 makeExtension.flags = {
-  name: '  The name of the extension. Preferable prefixed with `amp-*`',
+  name: '  The name of the extension. Should be prefixed with `amp-*`',
```

Comment:
@rcebulko I think the `--bento` change makes the prefix required, but correct me if I'm wrong.

Reply:
@alanorozco The `--bento` version actually accepts either `amp-foo-bar` or just `foo-bar` and treats them identically, so as to avoid confusion when people use it. Something like `const componentName = argv.name.replace(/^amp-/, '');`. I think that might be reasonable for this version too